.openshift/rocket-chat-ephemeral.json

```diff
@@ -262,6 +262,30 @@
                                         "mountPath": "/app/uploads"
                                     }
                                 ],
+                                "readinessProbe": {
+                                    "httpGet": {
+                                        "path": "/api/v1/info",
+                                        "port": 3000,
+                                        "scheme": "HTTP"
+                                    },
+                                    "initialDelaySeconds": 5,
+                                    "timeoutSeconds": 1,
+                                    "periodSeconds": 10,
+                                    "successThreshold": 1,
+                                    "failureThreshold": 3
```

Comment:
@jfchevrette I've been digging in the documentation recently and couldn't tell does the liveness check wait to start until after readiness check?

https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/

If it does wait, then I agree 15 seconds is more then enough.